apps/cc.gt

```diff
@@ -7,7 +7,9 @@ const vertices : vertexset{Vertex} = edges.getVertices();
 const IDs : vector{Vertex}(int) = 1;
 
 func updateEdge(src : Vertex, dst : Vertex)
-    IDs[dst] min= IDs[src];
+    if IDs[dst] != IDs[src]
```

Comment:
Could you revert the changes in this file. The if statement doesn't change the performance much and is a bit confusing. It's a test that I forgot to change back.